File: KendoPractice/src/main/java/com/tatvacoconet/service/IUserChandniService.java
package com.tatvacoconet.service;

import java.util.List;

import com.tatvacoconet.entity.UserMasterChandni;

public interface IUserChandniService {
	public List<UserMasterChandni> getUserList();
	public void deleteUser(Integer userId);
	public void createUser(UserMasterChandni user);
	public void updateUser(UserMasterChandni user);
	public UserMasterChandni find(Integer userId);
	public UserMasterChandni findUserByEmail(String email);
	@SuppressWarnings("rawtypes")
	public List getUsersByCity(String columnName);
}

File: KendoPractice/src/main/webapp/resources/app/Demo/Services/IUserChandniService.ts
/// <reference path='../../_allChandni.ts' />

module DemoChandni {
    export interface IUserChandniService {
        GetUserList: ($scope: ICOCOChandniScope) => ng.IPromise<Array<any>>;
        deleteUser: ($scope: ICOCOChandniScope, userId: any) => ng.IHttpPromise<string>;
        addUpdateUser: ($scope: ICOCOChandniScope, user: any) => ng.IHttpPromise<string>;
        FindUserByEmail: ($scope: ICOCOChandniScope, email: string) => ng.IHttpPromise<boolean>;
        GetUsersByCity: ($scope: ICOCOChandniScope) => ng.IHttpPromise<UserChandni>;
    }
}